pkg/preference/variables.go

```diff
@@ -60,13 +64,13 @@ const (
 )
 
 // TimeoutSettingDescription is human-readable description for the timeout setting
-var TimeoutSettingDescription = fmt.Sprintf("Timeout (in seconds) for OpenShift server connection check (Default: %d)", DefaultTimeout)
+var TimeoutSettingDescription = fmt.Sprintf("Timeout (in Duration) for cluster server connection check (Default: %s)", DefaultTimeout)
 
 // PushTimeoutSettingDescription adds a description for PushTimeout
-var PushTimeoutSettingDescription = fmt.Sprintf("PushTimeout (in seconds) for waiting for a Pod to come up (Default: %d)", DefaultPushTimeout)
+var PushTimeoutSettingDescription = fmt.Sprintf("PushTimeout (in Duration) for waiting for a Pod to come up (Default: %s)", DefaultPushTimeout)
```

Comment:
Surely nitpicking ^^, but would it be possible to change the format string? I found it a bit weird that this shows up as `4m0s` in the help message. Same thing for the default values of the other durations, except for `Timeout`:

```
❯ odo help preference                                                                                                                                                                                                                                                            
Modifies odo specific configuration settings within the global preference file.                                                                                                                                                                                                  
                                                                                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                                                 
Available Global Parameters:                                                                                                                                                                                                                                                     
 ConsentTelemetry - If true, odo will collect telemetry for the user's odo usage (Default: false)                                                                                                                                                                                
                    For more information: https://developers.redhat.com/article/tool-data-collection                                                                                                                                                                             
 Ephemeral - If true, odo will create an emptyDir volume to store source code (Default: true)                                                                                                                                                                                    
 PushTimeout - PushTimeout (in Duration) for waiting for a Pod to come up (Default: 4m0s)                                                                                                                                                                                        
 RegistryCacheTime - For how long (in Duration) odo will cache information from the Devfile registry (Default: 15m0s)                                                                                                                                                            
 Timeout - Timeout (in Duration) for cluster server connection check (Default: 1s)                                                                                                                                                                                               
 UpdateNotification - Flag to control if an update notification is shown or not (Default: true)                                                                                                                                                                     

```

Reply:
That format is how golang parses duration. I'll have to see if that can be modified, but another way would be to parse everything into `second`.